docs/source/release/v6.3.0/diffraction.rst

```diff
@@ -20,10 +20,10 @@ Improvements
 ############
 * Several improvements have been made to :ref:`AnvredCorrection <algm-AnvredCorrection>` (and :ref:`SphericalAbsorption <algm-SphericalAbsorption>` which calls it)
 
-  * the algorithm has been extended to evaluate the attenuation by more absorbing spherical samples (with muR < 9).
+  * the algorithm has been extended to evaluate the attenuation by more absorbing spherical samples (but with muR < 9).
```

Comment:
Sorry, I am finding this a bit unclear. Has this been extended to calculate the attenuation of spherical sample with high absorption coefficients? Does it have a max value of 9 for the absorption? What is the absorbing with respect to (this one is really for interest)?

Reply:
There was a change to make the calculation for better for high attenuation (which means high muR). Even with this change though there is a cap where it still doesn't work for really big muR. So the max muR it works for is 9.
I changed this because I felt the bit brackets perhaps suggested that that highly absorbing spherical samples had low muR and maybe muR>9 was low absorption.
Not sure I understand your last question - although Richard worked on the change so I may not be fully up to speed on the detail anyway

Reply:
my last question was what it absorbs; light, water, neutrons? 

Would the following be more clear:
The algorithm has been extended to allow for the calculation of the attenuation for spherical samples with muR < 9.

Reply:
It's neutrons.
Not really - the change specifically relates to spherical samples with a high muR (but still one that is less than 9). I don't know exactly but 5<muR<9 for example. So it needs to refer to high absorption

Reply:
Yes sure. I'll update. Also just noticed the threshold is actually 8 in the code and not 9 so I'll change that number as well. Will change to:
`the algorithm has been extended to evaluate the attenuation by more absorbing spherical (high muR) samples. Note - even with this change the calculation is still only accurate for muR < 8.`
I've just checked and the calculation still proceeds if muR>8 but the algorithm outputs a warning saying the results aren't accurate

Reply:
I think its the phrase "more absorbing" that is making me think this is a bit clunky. But, that is much clearer.

Reply:
Just tried rewording further but not really coming up with anything better so think I'll just push this change